src/System.Linq.Parallel/tests/PlinqModesTests.cs

```diff
@@ -11,157 +11,135 @@ namespace Test
 {
     public static class PlinqModesTests
     {
-        [Fact]
-        public static void RunPlinqModesTests()
+        private static IEnumerable<Labeled<Action<ParVerifier, ParallelQuery<int>>>> EasyUnorderedQueries()
```

Comment:
I believe "Easy" was chosen by the original author to denote queries that PLINQ would have an easy time parallelizing - so they'd be run that way by default (in contrast to hard, which might not).  I can add some explanatory comments; unless you have a better name?  I just stuck with the original, because I didn't really have anything better.